packages/suite/src/views/settings/debug/InvityApi.tsx

```diff
@@ -52,7 +52,9 @@ export const InvityApi = () => {
                         invityAPI.setInvityServersEnvironment(item.value);
                         // reload the Suite to reinitialize everything
                         if (typeof window !== 'undefined') {
-                            window.location.reload();
+                            window.setTimeout(() => {
+                                window.location.reload();
+                            }, 100);
```

Comment:
Not happy about this but it's in the Debug section, it's an easy fix and it gets the job done. 🤷‍♂️